amazonka-s3-streaming.cabal

```diff
@@ -45,18 +46,18 @@ source-repository head
 
 executable s3upload
   main-is: Main.hs
-  ghc-options: -threaded -rtsopts -with-rtsopts=-N
+  ghc-options: -threaded -rtsopts "-with-rtsopts=-N -qg"
   default-language: Haskell2010
   if flag(s3upload-exe)
-     buildable: True
+    buildable: True
   else
      buildable: False
-  build-depends: base
+  build-depends: base >= 4.9 && < 5
```

Comment:
Shouldn't have to repeat bounds because you have bounds on the `library`'s `build-depends`, but also please don't do `base >=4.x && <5` - I have hit an annoying number of packages which claim this and fail on GHC 9.0.1 (`base-4.15`) or GHC 9.2.1 (`base-4.16`).